### DISCUSSION TITLE
Geräte Einweisungsgruppen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#24] [Shipped - Live on production]
Geräte auswählen:

Geräte wählt man prinzipiell aus der Bestandsliste aus. Wenn dort nicht vorhanden alternativ Zugriff auf Katalog.

1. Aktuell 2 Wege für Zugriff auf globalen Katalog -> reduzieren
2. Wenn Gerät erst einmal hinzugefügt, ist nicht mehr ersichtlich woher es kommt -> Geräte aus dem Katalog oder Bestand markieren/ grafisch absetzen

=> Folge aktuell: User weiß, dass Gerät in seiner Gruppe ist > öffnet neuen Einweisungsvorgang > sucht nach Gerät > findet Gerät nicht, da aus öffentlichem Katalog

<img width="1510" alt="Bildschirmfoto 2022-08-17 um 16 40 21" src="https://user-images.githubusercontent.com/80519295/185173604-eaafc47b-f0a0-4475-86b1-5a63d75afe56.png">

**Comment by YvesR** [comment]
Den Teil mit "erstellt neue Einweisung, findet das Gerät nicht" musst Du noch mal erläutern.

Prinzipiell habe ich angedacht, dass man den Schieberegler entfernt und in der Direkt-Suche nur noch nach Typen im Inventar sucht. In dem bereits existierenden Dialog dann immer public (dort markiert, ob es auch als Inventar vorhanden ist, siehe anderes Issue).

**Reply by moritzvolkmar** [reply]
Damit war folgendes gemeint:
Ein Gruppenorganisator weiß ja, welche Geräte in seiner Gruppe sind. Bspw. eine **Evita 2**

![Bildschirmfoto 2022-09-19 um 13 47 46](https://user-images.githubusercontent.com/80519295/191011042-c4ca2cff-7cd4-4691-bc2f-dc8421fa8b73.png)

Die wurde irgendwann mal aus dem öffentlichen Katalog zur Gruppe hinzugefügt, ist aber nicht oder nicht mehr im Bestand.
Aus irgendeinem Grund will nun jemand nur auf diesen Typ eine Einweisung dokumentieren. Er sucht im Vorgang also nach **Evita 2** und findet nichts, da es kein Gerät diesen Typs im Bestand gibt.

![Bildschirmfoto 2022-09-19 um 13 48 52](https://user-images.githubusercontent.com/80519295/191011391-07d962b8-527a-4dd2-956e-1fb5120f0ea4.png)

Das Grundproblem ist glaube ich eher, dass nicht ersichtlich ist, wo man gerade sucht. Ob nur im Bestand oder im globalen Katalog.

**Comment by YvesR** [comment]
Durch neue Dialog-Logik Thema obsolet. Kommt somit im nächsten Relase indirekt als Feature.
